modules/storage-expanding-csi-volumes.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies
+//
+// * storage/expanding-persistent-volumes.adoc
+
+[id="expanding-csi-volumes_{context}"]
+= Expanding CSI volumes
+
+Support for expanding CSI volumes is enabled by default, but it also requires a specific CSI driver to support volume expansion.
+
+:FeatureName: Expanding CSI volumes
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+[IMPORTANT]
+====
+{product-title} does not ship with any CSI drivers. It is recommended
+to use the CSI drivers provided by
+link:https://kubernetes-csi.github.io/docs/drivers.html[community or storage vendors].
+
+Installation instructions differ by driver, and are found in each driver's
+documentation. Follow the instructions provided by the CSI driver.
+
+// OCP 4.2 supported CSI 1.1.0, so bumping OCP 4.3 to 1.2.0. Need to verify.
+{product-title} {product-version} supports version 1.2.0 of the
```

Comment:
@gnufied Will OCP 4.3 support CSI spec 1.2.0?

Reply:
No 4.3 will not support 1.2.0. 1.2.0 was just now tagged and PR to update k8s with CSI-1.2.0 is still outstanding.

Reply:
Changed to 1.1.0.